postman/Manage Custom Fields.postman_collection.json

```diff
@@ -0,0 +1,400 @@
+						"});",
+						"",
+						"pm.test(\"Successfully submitted request\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});"
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+					"exec": [
+						"pm.test(\"Successfully get a  random list\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						"pm.test(\"At least one list returned\", function() {",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.total_size).to.be.above(0);",
+						"    pm.globals.set(\"listId\",jsonData.entries[0].id );",
+						"});"
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/0/accounts/{{accountId}}/lists",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"0",
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get the First List",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "75df1ad8-7a66-40eb-b8e7-2991fcd51619",
+					"exec": [
+						"pm.test(\"Successfully get a list\", function(){",
+						"    pm.response.to.have.status(200);",
+						"});",
+						""
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Custom Fields",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "b5a97102-9d21-4722-a899-3444de45eb5b",
+					"exec": [
+						"var jsonData =  pm.response.json();",
+						"",
+						"pm.test(\"Get custom fields in list\", function(){",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						"pm.test(\"One or more custom fields returned\", function() {",
+						"    pm.expect(jsonData.total_size).to.be.above(0);",
+						"});",
+						"",
+						"pm.test(\"New custom field is not empty and not null\", function() {",
+						"   pm.expect(pm.environment.has(\"custom_field\")).not.eql(\"\") && pm.expect(pm.environment.get(\"custom_field\")).to.eql(true);",
```

Comment:
This will always fail. TIL `eql` tests for "deep equality" by comparing the properties of the two objects: https://stackoverflow.com/questions/36798993/what-is-the-difference-between-equal-and-eql-in-chai-library

Since one side of the equals is a string and the other is a bool, their properties will never be equal.